test/migration/verify/verify_migration_test.go

```diff
@@ -119,6 +122,43 @@ func verifySecondMemberProvisionedSpace(t *testing.T, awaitilities wait.Awaitili
 	cleanup.AddCleanTasks(t, awaitilities.Host().Client, userSignupForSpace)
 }
 
+func verifyProvisionedSubSpace(t *testing.T, awaitilities wait.Awaitilities) {
+	parentSpace, _ := VerifyResourcesProvisionedForSpace(t, awaitilities, migration.ProvisionedParentSpace)
+	userSignupForSpace := checkMURMigratedAndGetSignup(t, awaitilities.Host(), migration.ProvisionedParentSpace)
+	targetClusterRoles := []string{cluster.RoleLabel(cluster.Tenant)}
+	hostAwait := awaitilities.Host()
+	memberAwait := awaitilities.Member2()
+	memberCluster, found, err := hostAwait.GetToolchainCluster(t, cluster.Member, memberAwait.Namespace, nil)
+	require.NoError(t, err)
+	require.True(t, found)
+
+	subSpace, err := awaitilities.Host().WaitForSubSpace(t,
+		migration.ProvisionedSpaceRequest,
+		memberAwait.Namespace,
+		parentSpace.GetName(),
+		wait.UntilSpaceHasTargetClusterRoles(targetClusterRoles),
+		wait.UntilSpaceHasTier("base"),
+		wait.UntilSpaceHasAnyProvisionedNamespaces())
+	require.NoError(t, err)
+
+	spaceRequest, err := memberAwait.WaitForSpaceRequest(t,
+		types.NamespacedName{
+			Namespace: memberAwait.Namespace,
+			Name:      migration.ProvisionedSpaceRequest,
+		},
+		wait.UntilSpaceRequestHasConditions(wait.Provisioned()),
+		wait.UntilSpaceRequestHasStatusTargetClusterURL(memberCluster.Spec.APIEndpoint),
+		wait.UntilSpaceRequestHasNamespaceAccess(subSpace),
+	)
+	require.NoError(t, err)
+	VerifyNamespaceAccessForSpaceRequest(t, memberAwait.Client, spaceRequest)
+
+	cleanup.AddCleanTasks(t, awaitilities.Host().Client, subSpace)
+	cleanup.AddCleanTasks(t, awaitilities.Member1().Client, spaceRequest)
```

Comment:
minor detail - seems like you are using member2 for the creation of the spacerequest and not member1. Probably the cleanup will still work since it's there is only 1 cluster in e2e mode.

Reply:
yep, I'm intending on using `member2` for this test.

Reply:
Fixed in 1048498.